style.css

```diff
@@ -348,13 +345,9 @@ nav h2 {
   margin: 10% 0;
 }
 
-.date {
-  margin-top: 100px;
-}
-
 /* Main Program */
 .main-program {
-  background-color: #272a31;
+  background-image: url(./Assets/images/Program-bg.jpg);
```

Comment:
- [Optional] Great Job for updating the previous suggestions.👍  I think you missed the bg image spelling `Assets/images/program-bg.jpg` so it is better to use only small letters for the asset file👍👍